taca/analysis/analysis_nanopore.py

```diff
@@ -153,19 +151,29 @@ def process_qc_run(ont_qc_run: ONT_qc_run):
 
     # Is the run fully synced?
     if not ont_qc_run.is_synced():
-        logger.info(f"{ont_qc_run.run_name}: Run is not fully synced, skipping.")
-    else:
-        # Assert all files are in place
-        logger.info(f"{ont_qc_run.run_name}: Asserting run contents...")
-        ont_qc_run.assert_contents()
+        raise WaitForRun(f"{ont_qc_run.run_name}: Run is not fully synced, skipping.")
 
-        # Update StatusDB
-        logger.info(f"{ont_qc_run.run_name}: Updating StatusDB...")
-        ont_qc_run.update_db_entry()
+    # Assert all files are in place
+    logger.info(f"{ont_qc_run.run_name}: Asserting run contents...")
+    ont_qc_run.assert_contents()
```

Comment:
Missing files here would raise assertionErrors. I guess it could be presented in a nicer way to the user?

Reply:
Forget about it, you catch the exception nicely outside of the method.